src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -283,8 +283,9 @@
                 "downloadExercise": "Programmierübung herunterladen",
                 "downloadAllStudents": "Oder lade die Repositories aller Studierenden herunter",
                 "downloadAllTeams": "Oder lade die Repositories aller Teams herunter",
-                "filterLateSubmissions": "Verspätete Abgaben herausfiltern",
+                "filterLateSubmissions": "Abgaben aus dem Übungsmodus ausschließen",
```

Comment:
This change is not intended, right?

Reply:
Good catch. I confused myself by switching between GitHub comments and code :/ Thanks a lot! Will fix it